### PR TITLE
Web App settings for swipe or place rifd

### PR DESCRIPTION
**Pull request by MiczFlor**
See discussion at https://github.com/MiczFlor/RPi-Jukebox-RFID/pull/1122
This creates the settings pulldown in the web app under "Settings"

### CODE REVIEW

**Comment by s-martin** [comment]
@Toqqi, could you test this? As soon as this is tested, we can merge this.

File: htdocs/inc.setPlayerBehaviourRFID.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            print $exec; print "<br>Swipe_or_Place: ".$Swipe_or_Place;
+        }
+        exec($exec);
+    } 
+    // execute shell to create config file
+    $exec = "sudo ".$conf['scripts_abs']."/inc.writeGlobalConfig.sh";        
+    if($debug == "true") {
+        print $exec;
+    }
+    exec($exec);
```
Comment (Toqqi) [inline htdocs/inc.setPlayerBehaviourRFID.php:34]: Please add an RFID Reader Service restart, so the settings change from the web app can be used instantly.
```
    // execute shell to restart RFID Reader Service
    $exec = "sudo systemctl restart phoniebox-rfid-reader.service";
    if($debug == "true") {
        print $exec;
    }
    exec($exec);
```

Comment (s-martin) [reply htdocs/inc.setPlayerBehaviourRFID.php:36]: @Toqqi i have added your required change

**Comment by s-martin** [comment]
I merged it as @Toqqi tested it successfully, now we could continue at #1122?

**Comment by Kyokuhin-b** [comment]
I habe done the changes. What should change in the web app?

**Comment by Kyokuhin-b** [comment]
Hi, I got a new Menu " General RFID player control"  but it doesn't work

**Comment by s-martin** [comment]
This PR is only the web interface, logic is implemented in #1122

**Comment by Kyokuhin-b** [comment]
OK, then I got it right.

The web interface works great in any case!!!

I was just unsure because I had implemented the code from # 1122, but unfortunately it wasn't working properly yet.

Many Thanks

**Pull request merged**